cyflash/bootload.py

```diff
@@ -383,16 +383,30 @@ def check_metadata(self, data, downgrade, newapp, psoc5):
     def write_rows(self, data):
         total = sum(len(x) for x in data.arrays.values())
         i = 0
+        nbOfTries = 3
```

Comment:
variable names in this file should be `snakecase`.

Reply:
ok